src/Attribute/ISimple.php

```diff
@@ -32,20 +32,26 @@ interface ISimple extends IAttribute
      * Returns the SQL primitive type declaration in MySQL notation. i.e. "text NULL".
      *
      * @return string
+     *
+     * @deprecated Implement schema generators instead.
```

Comment:
We should add `See #1267.` here. Linking to this PR can be very meaningful for developers.